README.md

```diff
@@ -8,3 +8,267 @@ To enable one of the scripts you have to add a line like `require "official/hell
+
+
+### autostyle.lua
+
+*Compatibility: 1.x, 2.x*  
+*Maintainer: Marc Cousin (cousinmarc@gmail.com)*
+
+Automatically apply a given style when an exif tag is present in the file, e.g. to apply a style to compensate for Auto-DR from some Fujifilm cameras.
+
+
+### calcDistance.lua
+
+*Compatibility: 2.x*  
+*Maintainer: Tobias Jakobs*  
+*Tags: geo*
+
+Calculate the distance between the places where two images were taken by using the GPS metadata.
+
+
+### copy_attach_detach_tags.lua
+
+*Compatibility: 2.x, 3.x*  
+*Maintainer: Christian Kanzian*  
+*Tags: tagging*
+
+Copy, paste, replace and remove tags from images by shortcuts and a distinct lighttable module.
+
+
+### cr2hdr.lua
+
+*Compatibility: 1.x, 2.x*  
+*Maintainer: Till Theato (theato@ttill.de)*  
+*Dependencies: cr2hdr program from Magic Lantern (http://www.magiclantern.fm/forum/index.php?topic=7139.0)*
+
+Process images shot with Magic Lantern's Dual ISO feature for dynamic range improvement with the cr2hdr program from lighttable and import the result (with shortcut and/or on import).
+
+
+### geoJSON_export.lua
+
+*Compatibility: 2.x*  
+*Maintainer: Tobias Jakobs*  
+*Dependencies: mkdir, convert (ImageMagick), xdg-open, xdg-user-dir*  
+*Tags: geo*
+
+Generate GeoJSON file from image metadata. **What exactly is exported? What is a typical use case?**
+
+
+### geo_uri.lua
+
+*Compatibility: 2.x*  
+*Maintainer: Tobias Jakobs*  
+*Dependencies: gnome-maps ≥ 3.20*  
+*Tags: geo*
+
+Open a geo uri in gnome-maps. **Where is this URI found? What is a typical use case?**
+
+
+### gimp.lua
+
+*Compatibility: 2.x*  
+*Maintainer: Bill Ferguson (wpferguson@gmail.com)*  
+*Dependencies: GIMP*  
+*Tags: storage*
+
+Adds new export option to lounch gimp with the selectet photo. After editing in GIMP the result is imported back into Darktable.
+
+
+### gps_select.lua
+
+*Compatibility: 1.x, 2.x*  
+*Maintainer: Tobias Jakobs*  
+*Tags: geo*
+
+Select images with or without GPS information with shortcuts.
+
+
+### hugin.lua
+
+*Compatibility: 1.x, 2.x*  
+*Maintainer: Tobias Jakobs*  
+*Dependencies: Hugin*  
+*Tags: storage*
+
+Add a new storage option to send images to hugin.
+
+
+### kml_export.lua
+
+*Compatibility: 2.x*  
+*Maintainer: Erik Augustin*  
+*Dependencies: mkdir, zip, convert (ImageMagick), xdg-open, xdg-user-dir*  
+*Tags: geo, storage*
+
+Adds a new export option to export KML files. **I have no clue what it really does.**
+
+
+### rate_group.lua
+
+*Compatibility: unknown*  
+*Maintainer: Dom H (dom@hxy.io)*
+
+Provide shortcuts for rating or rejecting all images within a group.
+
+
+### slideshowMusic.lua
+
+*Compatibility: 1.x, 2.x*  
+*Maintainer: Tobias Jakobs*  
+*Dependencies: rhythmbox-client*
+
+Play music during a slide show.
+
+
+### video_mencoder.lua
+
+*Compatibility: 1.x*  
+*Maintainer: Tobias Jakobs*  
+*Dependencies: mencoder, xdg-open, xdg-user-dir*
+
+Video export from darktable. **Some more information would be great.**
```

Comment:
It creates a video from the selected images like this one (Sorry, the text is in German.):
http://dablogter.blogspot.co.uk/2014/04/timelapse-tiltshift-video.html 

I can't use this script any more, because mencoder is no longer part from Ubuntu. :( It should be ported to ffmpeg.